### DISCUSSION TITLE
Is there some way to custom the Overlay text?

### DISCUSSION THREAD

**Discussion by brupinto** [EveOPlus/eve-o-preview#19] [General]
Hello there,

Congratulations about project. It is so great!

I am speending some time setting the position to work in the same position of all pilots in the same account. Now I could will like know if we can custom the text of overlay I mean change the attributes of size and color.

Tank you to speend time to ansewer me.

fly safe 0/

**Comment by EveOPlus** [comment]
This is available in the latest releases.
